luarules/gadgets/team_transfer/api_widgets.lua

```diff
@@ -0,0 +1,4 @@
+local M = {}
+
```

Comment:
shouldn't this have something in it? The widget sharing helpers that already existed in common should go here and be factored in the module somewhere.